src/pages/LoadingPage/LoadingPage.jsx

```diff
@@ -13,8 +13,7 @@ const LoadingPage = ({ currentPage, setCurrentPage }) => {
       }, 2000);
       setPage(currentPage);
     }
-    if(page==="loading1")
-    {
+    if (page === "loading1") {
       setTimeout(() => {
         setCurrentPage("roadmap");
```

Comment:
As per the previous comment you can just be changing the page index instead of a string. I know it wasn't your code but this logic seems really broken.